### ISSUE TITLE
Suggestion for renaming this repository

### ISSUE THREAD

**Issue by cgendreau** [DINA-Web/dina-schema#25]
Even if a understand why this repository was named "schema", I would suggest to rename it to something more precise where we could guess by the name that it contains the "api documentation and json schemas for domain models".

**Comment by antonoberg** [comment]
Agree! I think that naming of our repositories in general need some love. Will bring it up tomorrow

**Comment by antonoberg** [comment]
There is also plans to merge this repository into our ui repo dina-collections-ui.
